api/GlobalEventHandlers.json

```diff
@@ -4503,10 +4503,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/GlobalEventHandlers/ontouchcancel",
           "support": {
             "chrome": {
-              "version_added": "18"
+              "version_added": false
```

Comment:
This was changed from "18", but confirmed by the comment and flag at https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/platform/runtime_enabled_features.json5;l=1986;drc=2e80c2498eb1d8c67b36ee6b27fa6381eb8aacab